src/redux/sagas/start.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+//
+// (c) 2019 Jeremy Green
+
+import { put, takeEvery } from 'redux-saga/effects';
+
+import { START } from '../actionTypes';
+import { levelChange, resetScore, screenChange } from '../actions';
+
+export function* start(): Generator<*,*,*> {
+  yield put(resetScore());
+  yield put(levelChange(1));
+  yield put(screenChange("start"));
+}
```

Comment:
This new saga is used when first run the app, and if restart. Added new blank "start" screen that triggers this saga when first run the app.